core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -100,6 +101,7 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
       pollExpected = true
       poll()
     case RequestMessages(topics) =>
+      context.watch(sender())
```

Comment:
As I wrote before (https://github.com/akka/reactive-kafka/pull/139#discussion_r68251195) I think we better will get an extra `Termination` message rather that constantly doing `watch`/`unwatch` for the same `Sources` for each message requests. I am pretty sure that there is relatively small number of sources which uses the same connection. What do you think?